### PR TITLE
swat: oranges pleaseibegyou edition

### PR DESCRIPTION
**Pull request by Bawhoppen**
same pr as https://github.com/tgstation/tgstation/pull/20355 except 
# DOES NOT BUFF SWAT ARMOR JUST ADDS THE TACTICOOL VARIANT OF IT FOR CONSISTENCY

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
So why are you buffing swat?

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -99,7 +99,7 @@
 	desc = "An extremely robust, space-worthy helmet in a nefarious red and black stripe pattern."
 	icon_state = "swatsyndie"
 	item_state = "swatsyndie"
-	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20, fire = 50, acid = 50)
+	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20, fire = 100, acid = 100)
```
Comment (ChangelingRain) [inline code/modules/clothing/head/helmet.dm]: good sir, good sir!

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -404,4 +404,9 @@
 	icon_state = "knight_templar"
 	item_state = "knight_templar"
 
-
+/obj/item/clothing/suit/armor/swat
+	name = "\improper SWAT vest"
+	desc = "A lightweight assault SWAT vest."
+	icon_state = "swatvest"
+	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20, fire = 100, acid = 100)
```
Comment (ChangelingRain) [inline code/modules/clothing/suits/armor.dm]: good sir this armor is stronger, good sir,

**Comment by optimumtact** [comment]
You're breaking the camera

File: code/modules/clothing/suits/armor.dm (hunk 2 above)
Comment (Bawhoppen) [inline code/modules/clothing/suits/armor.dm]: No it isn't, swat suits already had that, I just made it so they all had that consistently

**Comment by Bawhoppen** [comment]
They are not buffed, just consistent now

**Comment by Cheridan** [comment]
dude really

**Pull request closed without merging**